server/controllers/auth.controller.js

```diff
@@ -167,8 +164,8 @@ function loginWithEmail(req, res, next) {
  */
 
 function register(req, res, next) {
-  const username = req.body.username;
-  const password = req.body.password;
+  const { username } = req.body;
```

Comment:
This destructuring can be done in one line:

const { username, password } = req.body;